nested/webvmss.json

```diff
@@ -7,6 +7,12 @@
                 "description": "Common Moodle values"
             },
             "type": "object"
+        },
+        "storageAccountKey": {
+            "metadata": {
+                "description": "Key used to create the Azure Files share in case azurefiles is specified as fileServerType"
```

Comment:
Should probably remove Azure Files reference from description.

Reply:
Rephrased